cinder-{{cookiecutter.driver_name_lc}}/src/lib/charm/openstack/cinder_{{cookiecutter.driver_name_lc}}.py

```diff
@@ -1,37 +0,0 @@
-import charms_openstack.charm
-import charmhelpers.core.hookenv as ch_hookenv  # noqa
-
-charms_openstack.charm.use_defaults('charm.default-select-release')
-
-
-class Cinder{{ cookiecutter.driver_name }}Charm(
-        charms_openstack.charm.CinderStoragePluginCharm):
-
-    # The name of the charm
-    name = 'cinder_{{ cookiecutter.driver_name_lc }}'
-
-    # Package to determine application version. Use "cinder-common" when
-    # the driver is in-tree of Cinder upstream.
-    version_package = '{{ cookiecutter.additional_package_name|default("cinder-common", true) }}'
-
-    # Package to determine OpenStack release name
-    release_pkg = 'cinder-common'
-
-    # this is the first release in which this charm works
-    release = '{{ cookiecutter.release }}'
-
-    # List of packages to install
-    packages = ['{{ cookiecutter.additional_package_name }}']
```

Comment:
Same for this one, we still want to provide the example of how to install packages?

Reply:
Fixed. thanks